OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -245,7 +245,7 @@ public override IEnumerable<TargetLineNode> TargetLineNodes(Actor self)
 					yield return n;
 
 			if (orderLocation != null)
-				yield return new TargetLineNode(Target.FromCell(self.World, orderLocation.Value), Color.Green);
+				yield return new TargetLineNode(Target.FromCell(self.World, orderLocation.Value), Color.Red);
```

Comment:
This should be `Color.Crimson` instead of `Color.Red` to be consistent with the other "red" target lines.